applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc

```diff
@@ -8,7 +8,7 @@ include::_attributes/servicebinding-document-attributes.adoc[]
 toc::[]
 
 [role="_abstract"]
-In addition to grouping multiple components within an application, you can also use the *Topology* view to connect components with each other. You can either use a binding connector or a visual one to connect components.
+In addition to grouping multiple components within an application, you can also use the *Topology* view to connect components with each other and to connect multiple resources to services with labels. You can either use a binding connector or a visual one to connect components.
```

Comment:
@divyanshiGupta WDYT?

Reply:
I'm using the second suggestion provided by @Srivaralakshmi. Instead of using the "visual one", I'll use the term "visual connector" which is specified later in the document.